mutator.go

```diff
@@ -236,14 +236,7 @@ func (m *Mutator) clusterBomb(template string, results chan string) {
 	// instead of sending all payloads only send payloads that are used
 	// in template/statement
 	for _, v := range varsUsed {
-		payloadSet[v] = []string{}
-		for _, word := range m.Options.Payloads[v] {
-			if !strings.Contains(template, word) {
-				// skip all words that are already present in template/sub , it is highly unlikely
-				// we will ever find api-api.example.com
-				payloadSet[v] = append(payloadSet[v], word)
-			}
-		}
+		payloadSet[v] = append([]string{}, m.Options.Payloads[v]...)
```

Comment:
Instead of removing the `!strings.Contains` condition that's already in place, it would be better to enhance it by performing a comparison check where if the `word` is eq to `sub`, then just `continue`.

Also don't forget to add/update the test case.